addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -286,7 +286,7 @@
 					<control type="label">
 						<width>$PARAM[ruler_label_width]</width>
 						<height>45</height>
-						<font>font12</font>
+						<font>font13</font>
```

Comment:
i wouldn't increase this one.
depending on your language & regional settings, the date label will be cropped as it's too wide

![date](https://user-images.githubusercontent.com/687265/35186156-b98dfd64-fe0f-11e7-8cc2-0e80b0792304.jpg)

Reply:
good point, will reverse